### PR TITLE
Profile picture of logged user and bugfixes

### PR DESCRIPTION
**Pull request by Ventan00**
* Added profile picture of logged user
* Added commented mock user for testing purposes
* Fixed Mozilla bug for logo scaling
* Removed element causing white field under footer

### CODE REVIEW

File: Client/Shared/TopBarComponent.razor [hunk 1]
```diff
@@ -38,4 +41,15 @@
     [CascadingParameter]
     protected bool isWhiteTheme { get; set; }
 
+    /// <summary>
+    ///     Zmienna przechowująca watar
```
Comment (Patrykoss) [inline Client/Shared/TopBarComponent.razor]: Literówka summary Avatar

**Review by Patrykoss — CHANGES_REQUESTED** [review]
Small typo

**Review by pnarozniak — COMMENTED** [review]
Everything is ok, except typo already mentioned above

**Review by Patrykoss — APPROVED** [review]
Ok

**Review by pnarozniak — APPROVED** [review]

**Pull request merged**